src/mixins/AddShareRequestByTemplate.ts

```diff
@@ -14,55 +15,62 @@ export function AddShareRequestByTemplate<TBase extends ConnectorTUIBaseConstruc
       const name = process.env.CONNECTOR_DISPLAY_NAME ?? "ConnectorV2 Demo"
```

Comment:
This seems to be outdated, `"ConnectorV5 Demo"` should now be written here as default `CONNECTOR_DISPLAY_NAME`.